### ISSUE TITLE
drag item(s) to JEI panel for delete - not working

### ISSUE THREAD

**Issue by smmmadden** [mezz/JustEnoughItems#1350]
Following the directions here:
With Cheat Mode Active:
Get a full stack of Items: Click Item
Get one Item: Right-Click Item
**Delete Item: Drop an Item into the Item List**
I try dragging one or more items from my inventory to the JEI panel and I don't see "Delete" but instead it displays the title of the block I'm hovering over.  I did see this working at one point while in and not in cheat mode.

Forge Build: 2759 (1.12.2)
Minecraft Version: 1.12.2
JEI Version: 1.12.2-4.12.0.214 (on server & client mods folder)
![image](https://user-images.githubusercontent.com/10309346/44931728-a8fd2d00-ad30-11e8-909e-8ba322c38647.png)
![image](https://user-images.githubusercontent.com/10309346/44931736-ac90b400-ad30-11e8-8a89-44fc763948ca.png)
![image](https://user-images.githubusercontent.com/10309346/44931747-b0bcd180-ad30-11e8-9c54-5b7c2b0a6a30.png)

**Scenarios tested:** 
Creative Mode - Cheat Enabled & Not Enabled
Survival Mode - Cheat Enabled & Not Enabled
Doesn't appear to be an option to delete any more and I don't see any config setting to enable it.
Thanks -Steve

**Comment by mezz** [comment]
Thanks for the report!
I have tried to reproduce this issue but I wasn't able to make it happen.
Can you remove other mods and try with just JEI? I think another mod may be involved here.

**Comment by smmmadden** [comment]
sure, I'll give it a go and let you know.

**Comment by smmmadden** [comment]
update: I reset my minecraft local folder, copied all the mods back to the client mods folder and tried again.  Some of the blocks that I tried above were not working without Cheat mode enabled.  I reenabled cheat mode and still had a problem with a couple other items but those were from another mod.  So the bug isn't with JEI as you suggested, it's with the integration to JEI from that mod not allowing the delete to occur.  This can be closed, thanks! -Steve
